guides/introduction/welcome.md

```diff
@@ -35,25 +35,27 @@ The query parameters are as follows.
 |-----------------|--------------------------|-----------|--------------------------------------------------------|
 | `_format`       | swiftui, jetpack, html   | ✅        | The content type to be processed by LiveView Native    |
 | `_interface`    | mobile, watch, tv        |           | The general device type                                |
-| `os_version`    | `string`                 |           | The version of the client OS                           |
-| `app_version`   | `string`                 |           | The version of the client application                  |
 
-This is formatted as `/?_format=xx&_interface=xx&os_version=xx&app_version=xx`, and when no query parameters are provided,
+This is formatted as `/?_format=xx&_interface=xx`, and when no query parameters are provided,
 will default to the corresponding Phoenix LiveView route (presuming a route is provided).
 
 ### Processing
 
 Once a request is sucessfully delegated by LiveView Native, it will attempt to match on your LiveView route.
 
-By design, LiveView Native does not ship with a default rendering package, and instead seperates
+By design, LiveView Native does not ship with a client, and instead seperates
 itself into a series of distinct packages. Each package ships with its own modifiers to handle its respective client,
 and unlike many framework agnostic development frameworks, intentionally seperates your markup by platform.
 
 This is to maintain instant feature parity with your platform(s) of choice, as LiveView Native is not concerned with
-cross-platform "bridging." This allows LiveView Native to serve native code to the platform target
-with no changes to the underlying client.
+a cross-platform abstraction layer (or an application bridge). This allows LiveView Native and its corresponding markup processor
+to send back a UI representation of your application.
 
-So long as your client supports the underlying components sent back, it will render!
+So long as your client supports the underlying markup sent back, it will render!
```

Comment:
I would note that each client should support the ability to extend itself through addons. For example, SwiftUI's client has several addons already (MapKit, RealityKit, PhotoKit, etc...) where we register certain views and modifiers for use via LVN. Additionally, we support the ability to add your own custom markup and modifiers (soon events). We're also working to make this much easier to do: https://github.com/liveview-native/liveview-client-swiftui/pull/1553